docs/setup-cluster/k8s/setup-multiple-resource-managers.rst

```diff
@@ -24,6 +24,10 @@ service in one Kubernetes cluster and schedule workloads in the same or other Ku
 Any requests to resource pools not defined in the master configuration are routed to the default
 resource manager. Such requests are not routed to additional resource managers, if defined.
 
+If you'd like to enable use of Determined tasks that rely on Determined proxies in the
+external-to-master clusters you'd need to set up a gateway as described in the docs :doc:`Internal
+Task Gateway here <internal-task-gateway>`.
```

Comment:
To enable use of Determined tasks that rely on Determined proxies in the
external-to-master clusters, set up a gateway as described in the docs :doc:`Internal
Task Gateway here <internal-task-gateway>`.

Reply:
we want to signal that this is optional and they would want to set it up if only the user needs to run these tasks on those external clusters.